NEWS.md

```diff
@@ -12,6 +12,18 @@ This release adds a number of helper functions, primarily for use with NONMEM mo
 
 * Per guidance in `bbr 1.0.0` release, `replace_tags()`, `replace_bbi_args()`, `replace_based_on()`, `add_decisions()`, and `replace_decisions()` have been removed.
 
+* Added `nm_file()`, `nm_grd()`, `nm_ext()`, `nm_tab()`, `nm_par_tab()`, and `nm_data()` for reading in NONMEM files more easily. (#426)
+
+* Deprecated `check_nonmem_table_output()`, `check_grd()`, and `check_ext()` and
+replaced them with `nm_file()` variants. These functions will warn about this
+for two more releases and then begin to error for two more releases before being
+removed altogether. (#426)
+
+* Deprecated `plot_nonmem_table_df()`, `plot_grd()`, and `plot_ext()` in an
+effort to more tightly define the scope of `bbr`. These functions will warn
+about this for two more releases and then begin to error for two more releases
+before being removed altogether. (#426)
```

Comment:
I'm not familiar with what the policy/practice is for bbr, but I was expecting this to say that these would be removed in a major release step (i.e. SemVer X.y.z bump).

Reply:
This is what we've done in the past, based on `tidyverse` practices. In practice, I don't think these functions are getting much use, so I'd be surprised if many (or any) users even notice.

Reply:
> This is what we've done in the past, based on `tidyverse` practices.

Okay, thanks for the context.  I'll keep that in mind (even if I think it's worth trying to adhere closer to semantic versioning rules/expectations than tidyverse practices).